packages/compiler/src/output/output_ast.ts

```diff
@@ -994,17 +1001,22 @@ export abstract class Statement {
   abstract visitStatement(visitor: StatementVisitor, context: any): any;
 
   hasModifier(modifier: StmtModifier): boolean {
-    return this.modifiers!.indexOf(modifier) !== -1;
+    return this.modifiers.indexOf(modifier) !== -1;
+  }
+
+  addLeadingComment(leadingComment: LeadingComment): void {
+    this.leadingComments = this.leadingComments || [];
+    this.leadingComments.push(leadingComment);
```

Comment:
For future fancyness and making the code extra hard to read, this could be

```suggestion
    (this.leadingComments ??= []).push(leadingComment);
```

But please don't 😱 (probably won't even fly in g3, as I believe this is TS 4.0 and g3 might still be on 3.9?)

But at least it could be

```suggestion
    this.leadingComments = this.leadingComments ?? [];
    this.leadingComments.push(leadingComment);
```

Reply:
Done